Application/src/Server_connection.java

```diff
@@ -0,0 +1,92 @@
+
+import java.io.*;
+import java.net.*;
+
+
+/**
+ * @Author Tim Jonasson
+ * @Version 1.0
+ * Handles the connection to the backend server
+ *
+ */
+
+public class Server_connection {
+    private static Socket socket = null;
```

Comment:
Why do are these static? Do you want multiple instances that run from the same socket? Why not use a singleton implementation?